docs/Tutorials/GettingStarted/Tutorial.md

```diff
@@ -195,7 +195,7 @@ This happens any time the "install" or "build" command are run on a deployment.
 
 The user can also build a component or deployment for a cross-compile by specifying the toolchain.  A previous generate
 for that toolchain should have been run. Again for deployments, the user typically should run "install", see below. These
-steps require the setup describe here: [RPI](https://github.com/nasa/fprime/blob/master/RPI/README.md)
+steps require the setup described here: [RPI](../../../RPI/README.md)
```

Comment:
These docs are rendered as part of our documentation.  Links like this that leave the `docs` directory need to be a full url to the github path.